shell.nix

```diff
@@ -0,0 +1,9 @@
+with (import <nixpkgs> {});
+stdenv.mkDerivation {
+  name = "srcEnv";
+  nativeBuildInputs = [
+                        tmux
```

Comment:
I think you want buildInputs instead of native here. Native is build time only. I'd also recommend adding tmux to the path in the script. (see demo default.nix in cardano-sl for how to generate path from deps).

Reply:
did it.